### ISSUE TITLE
Open link just clicking

### ISSUE THREAD

**Issue by jlesquembre** [kovidgoyal/kitty#371]
I hope this is the correct place for a question.

I know there is a `open_url_modifiers` option, but is possible to disable it? I want to open a link by just clicking on it, without pressing any modifier.

**Comment by kovidgoyal** [comment]
I'm afraid not. Terminal programs can grab the mouse and clicking is used for selections if they haven't grabbed the mouse. If you find it cumbersome to press amodifier and click the mouse at the same time, you might like the URL hints mode which allwos you to click URLS without needing to use the mouse at all.

**Comment by Limero** [comment]
I miss this feature as well, so I have created a pull request for it.
https://github.com/kovidgoyal/kitty/pull/504

**Comment by mschwartz** [comment]
This is not a problem.  In other terminal windows, no matter what the program running, I'm able to click on links.

In fact, the worst thing that happens is I didn't intend to click on the link, just on the window to activate it.

As it is, I don't easily remember the combination of keys required to click on a link and end up using google to find it.  And that's how I found this issue.

**Comment by bartenra** [comment]
Should be ctrl+click at least. :)

**Comment by SolitudeSF** [comment]
> Should be ctrl+click at least. :)

you can choose whatever modifiers you want

**Comment by Neurognostic** [comment]
Alternatively, instead of clicking on a URL, use the [`hints`](https://sw.kovidgoyal.net/kitty/kittens/hints.html) kitten.

```conf
map kitty_mod+e kitten hints
```

**Comment by Luflosi** [comment]
This is no longer the case in the latest version. See https://sw.kovidgoyal.net/kitty/conf.html#shortcut-kitty.Click-the-link-under-the-mouse-cursor-when-no-selection-is-created for example.

**Comment by Tyilo** [comment]
My kitty seems to have been updated and now clicking links without any modifier keys opens the link.

Is there a way of disabling this? I only want to open the link when clicking it while pressing ctrl+shift.

**Comment by kovidgoyal** [comment]
https://sw.kovidgoyal.net/kitty/conf/#mouse-actions

**Comment by jldevezas** [comment]
Took me a while to find, but it's in the manual:
`mouse_map left click ungrabbed no-op`

**Comment by githorse** [comment]
For consistency with most other terminal emulators, I'd like to map this to `ctrl+click` instead of `ctrl+shift+click`. I thought this line in `kitty.conf` would do it:

```
mouse_map ctrl+left release grabbed,ungrabbed mouse_handle_click link
```

But my mapping is still unchanged. How do I remap to `ctrl+click`?

**Comment by valerykh** [comment]
TLDR; now you can add `open_url_with default` to your config and it opens URL on mouse click

**Comment by hacker1024** [comment]
I open links with CTRL+Click like so:

```
mouse_map left click ungrabbed mouse_handle_click prompt
mouse_map ctrl+left click ungrabbed mouse_handle_click link
```

**Comment by jondkinney** [comment]
Is this supposed to be working on the latest kitty? I have 0.36.0 and I still have to hold modifiers to open links 🤔
